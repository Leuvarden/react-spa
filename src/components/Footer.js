import React from 'react';
import Logo from './Logo';

export default function Footer () {
    return (
        <footer>
            <div className="footer-container">
                <Logo />
            </div>
        </footer>
    );
}