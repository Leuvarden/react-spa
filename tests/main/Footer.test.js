import React from 'react';
import Footer from './../../src/components/main/Footer.js';

/* eslint-disable no-undef */
describe ('Footer', () => {

    it('renders as expected', () => {
        const footer = shallow(<Footer />);
    
        expect(footer).toMatchSnapshot();
    });

});