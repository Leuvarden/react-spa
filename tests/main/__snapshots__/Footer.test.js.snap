// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders as expected 1`] = `
<footer>
  <div
    className="footer-container"
  >
    <Logo />
  </div>
</footer>
`;
